packages/edit-site/src/components/style-book/index.js

```diff
@@ -46,20 +53,44 @@ const {
 const { mergeBaseAndUserConfigs } = unlock( editorPrivateApis );
 
 const { Tabs } = unlock( componentsPrivateApis );
-
+const noop = () => {};
 function isObjectEmpty( object ) {
 	return ! object || Object.keys( object ).length === 0;
 }
 
+const scrollToSection = ( anchor, iframe ) => {
+	if ( ! iframe ) {
+		return;
+	}
+	const frameWindow = iframe.contentWindow;
+	const element = frameWindow.document.getElementById( anchor );
+	if ( element ) {
+		element.scrollIntoView( {
+			behavior: 'smooth',
+		} );
+	}
+};
+
+const getStyleBookNavigationFromPath = ( path ) => {
+	if ( path && typeof path === 'string' && path.includes( '/blocks/' ) ) {
+		const block = decodeURIComponent( path.split( '/blocks/' )[ 1 ] );
+		return {
+			block,
```

Comment:
Returns an object to leave open the possibility of deep linking to `category > block` later.